src/main/java/org/jitsi_modified/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -564,13 +628,57 @@ public static void setKeyFrame(byte[] input, int offset, boolean keyFrame)
         //
         // 16 bits      :     (2 bits Horizontal Scale << 14) | Width (14 bits)
         // 16 bits      :     (2 bits Vertical Scale << 14) | Height (14 bits)
+        public static final byte[] START_CODES = {(byte) 0x9D, 0x01, 0x2A};
+
+        /**
+         * @return True, if VP8 Payload contains start codes of a keyframe, otherwise false.
+         */
+        public static boolean validate(byte[] buf, int off)
```

Comment:
This should take a `len` argument rather than relying on `buf.length` -- in the idioms used by jitsi-media-transform, the valid range of the packet can be shorter than buffer's allocated length.

Reply:
True, will update